src/controllers/expense.controller.js

```diff
@@ -0,0 +1,101 @@
+const {
+  getFilteredExpenses,
+  addExpense,
+  getExpenseById,
+  getExpenseByIdIndex,
+  removeExpenseById,
+  updateExpenseById,
+} = require('../services/expenses.service');
+
+const { getUserById } = require('../services/users.service');
+
+const getFilteredExpense = (req, res) => {
+  const { userId, from, to, categories } = req.query;
```

Comment:
Consider creating a folder for middleware and validate the user request before passing it to the controller. Also, you can throw the 400 error here